core/src/main/java/io/aiven/klaw/model/response/ResourceOverviewAttributes.java

```diff
@@ -6,7 +6,6 @@
 @Data
 public class ResourceOverviewAttributes {
   @NotNull boolean topicExists;
-  @NotNull boolean schemaExists;
```

Comment:
thinking how about keeping it here only and display in topic overview somewhere, so user would not have to click on the schema tab

Reply:
To clarify you would prefer to keep it here?

Reply:
yes